Platform/AlderlakeBoardPkg/Library/Stage1BBoardInitLib/EcSupport.c

```diff
@@ -348,7 +348,10 @@ GetBoardId (
   //
   *PlatformId = 0xFF;
 
+#if !defined(PLATFORM_ADLN) && !defined(PLATFORM_ASL)
```

Comment:
SBL config data is preferred if it is available. For this case, prefer using SiCfgData->EcAvailable.

Reply:
@gdong1 BoardId is required before SBL can read config data, 

at the point SBL reading boardId value via EC, the dlf flag is not available(will read platform ID 0 value)

Reply:
oh, yes.  PlatformId is not got yet.